connexion/decorators/produces.py

```diff
@@ -76,7 +77,12 @@ def dumps(data):
         """ Central point where JSON serialization happens inside
         Connexion.
         """
-        return "{}\n".format(json.dumps(data, indent=2))
+        if six.PY2 == True:
+            encoding = "utf-8"
+        else:
+            encoding = None
+
+        return "{}\n".format(json.dumps(data, indent=2, encoding=encoding))
```

Comment:
@rafaelcaricio,

you have suggested return "".format(json_content) 
but why are we not doing  return "{}\n".format(json_content) as it was being returned originally ?